http-protocol-binding.md

```diff
@@ -147,6 +147,14 @@ If a receiver detects the CloudEvents media type, but with an event format that
 it cannot handle, for instance `application/cloudevents+avro`, it MAY still
 treat the event as binary and forward it to another party as-is.
 
+When the `Content-Type` header is not prefixed with the CloudEvents media type,
+being able to distinguish between a _binary_ CloudEvent and a non-CloudEvent
+message can be a challenge. While this specification can not mandate that
+senders do not include any of the CloudEvents HTTP headers when the message
+is not a CloudEvent, it would be a reasonable for a receiver to assume that if
+the message has all of the mandatory CloudEvents attributes as HTTP headers
```

Comment:
What if it doesn't have all of the mandatory fields? Should be considered non-event or not valid event? Maybe is it more convenient to state: if a request has an id attribute, then it should be handled as event (even not valid one)

Reply:
If the incoming message doesn't have all (4) required attributes then it is not a valid CE. While SDKs might choose to treat it as a CE but flag it as "incomplete", that is there choice. But from a spec perspective I don't think we can say anything other than all 4 are required to be a CE, anything else isn't a CE.